simulation/test_torrent_status.cpp

```diff
@@ -76,35 +99,82 @@ TORRENT_TEST(status_timers)
 			// to stay within that range
 			if (ticks > 20 * 60 * 60)
 			{
-				// unfortunantly we can't test this many ticks
-				TEST_ERROR("This line should be reached but doesn't");
 				return true;
 			}
 
-			if (ticks == 0){
-				// torrent get ready for seeding on first tick, which means +1s
-				startseed_time = lt::clock_type::now();
+			if(tick_is_in_active_range){
+				// 1 second per tick
+				active_time++;
 			}
 
+			// resume on a tick which is far away from next tick check intervall
+			// because with very low upload limit it takes some time to unchock
+			switch(ticks)
```

Comment:
I don't do anything with the ticks_intervall because it does not speed up the simulation. Maybe because the download nodes still use 500ms ticks. That means the test use the default 500ms tick size.
I do stop and start the torrent before and after critiacal ticks to check them with less active torrent ticks because ticks with download and upload actions are quite expensive in terms of execution time. Running through 70000 paused torrent ticks take already 2/3 of the test exeution time and 1/3 of the execution time is used for 700 active ticks.
I could comment every case if thes existing case comments does already help to understand that.

If you mean ticks_check_intervall, this is just for prevent spamming assert errors into log, it does not save much execution time